requirements.txt

```diff
@@ -27,21 +30,29 @@ jupyter-server==1.3.0
 jupyterlab==3.0.3
 jupyterlab-pygments==0.1.2
 jupyterlab-server==2.2.0
+kiwisolver==1.3.1
 MarkupSafe==1.1.1
+matplotlib==3.4.1
 mistune==0.8.4
+monty==2021.3.3
+mpmath==1.2.1
 nbclassic==0.2.6
 nbclient==0.5.2
 nbconvert==6.0.7
 nbformat==5.1.2
 nest-asyncio==1.5.1
+networkx==2.5.1
 notebook==6.2.0
 numpy==1.20.1
 packaging==20.9
+palettable==3.3.0
 pandas==1.1.4
 pandocfilters==1.4.3
 parso==0.8.1
 pexpect==4.8.0
 pickleshare==0.7.5
+Pillow==8.2.0
+plotly==4.14.3
```

Comment:
What did this dependency come with?

Reply:
Scipy comes from ASE, the rest come from Pymatgen and its dependencies

Reply:
Error log from pip when installing requirements.txt:
```ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pymatgen 2021.2.16 requires monty>=3.0.2, which is not installed.
pymatgen 2021.2.16 requires networkx>=2.2, which is not installed.
pymatgen 2021.2.16 requires palettable>=3.1.1, which is not installed.
pymatgen 2021.2.16 requires plotly>=4.5.0, which is not installed.
pymatgen 2021.2.16 requires ruamel.yaml>=0.15.6, which is not installed.
pymatgen 2021.2.16 requires spglib>=1.9.9.44, which is not installed.
pymatgen 2021.2.16 requires sympy, which is not installed.
pymatgen 2021.2.16 requires uncertainties>=3.1.4, which is not installed.```